esp/esp/themes/theme_data/circles/templates/main.html

```diff
@@ -113,7 +113,7 @@
 </div>
 
 {% load preview %}
-{% miniblog_for_user None as announcements 6 %}
+{% miniblog_for_user AnonymousUser as announcements 6 %}
```

Comment:
On my second attempt at figuring out what miniblog_for_user does, nope, this is fine. I'm going to also take out the checking for "None" from that code since it won't work anymore and we don't want people trying to use it.